webpage.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<head>
+<title>Job Seeker</title>
+<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
Can you please download the CSS files, place them in a folder and refer to them, instead of using CDN?
- Because it might happen that those CDN links would be removed or break sometime in the future.
- Also, it will allow us to render the page while not concerning of being connected to the internet.

Reply:
Downloaded the css file